src/main/resources/io/jenkins/plugins/opentelemetry/steps/CreateSpanStep/config.jelly

```diff
@@ -0,0 +1,6 @@
+<?jelly escape-by-default='true'?>
```

Comment:
Still wip to support the pipeline syntax snippet generation

Reply:
Solved with https://github.com/jenkinsci/opentelemetry-plugin/pull/84/commits/ba5e8ae96d71183440d51804622b44d98422ca92